src/Mvc/test/WebSites/BasicWebSite/RazorComponents/FetchData.razor

```diff
@@ -50,4 +50,4 @@ else
 
         forecasts = await ForecastService.GetForecastAsync(StartDate);
     }
-}
+}
```

Comment:
Nit: revert?

Reply:
Actually I see in a lot of these cases you're removing the trailing linebreak. If it's deliberate please ignore this comment. It doesn't really bother me either way; just trying to minimize future noise with people adding it back.

Reply:
I don't know that we really have a policy for or against this. It's not something that I'm purposely doing.